aws_xray_sdk/ext/psycopg/patch.py

```diff
@@ -12,27 +10,17 @@ def patch():
         'connect',
         _xray_traced_connect
     )
-    wrapt.wrap_function_wrapper(
-        'psycopg.extensions',
-        'register_type',
-        _xray_register_type_fix
-    )
-    wrapt.wrap_function_wrapper(
-        'psycopg.extensions',
-        'quote_ident',
-        _xray_register_type_fix
-    )
 
     wrapt.wrap_function_wrapper(
-        'psycopg.extras',
-        'register_default_jsonb',
-        _xray_register_default_jsonb_fix
+        'psycopg_pool.pool',
+        'ConnectionPool._connect',
+        _xray_traced_connect
```

Comment:
Can you give some insight into why these specific functions are being wrapped? I see a new dependency on psycopg pool that was not there in the initial commit, what is this package responsible for and is it sufficient to only patch the connection?